docs/index.rst

```diff
@@ -26,10 +26,10 @@ Web
 The easiest way to get started with tracing is to instrument your web server.
 We support many `Web Frameworks`_. Install the middleware for yours.
 
-Databases
-~~~~~~~~~
+Instrument modules
+~~~~~~~~~~~~~~~~~~
 
-Then let's patch all the widely used Python libraries that you are running::
+Then let's patch all used Python libraries that you are running through::
```

Comment:
this doesn't really make sense:

`Then let's patch widely used Python libraries::`